zvedamrance/src/main/java/ServletDelete.java

```diff
@@ -0,0 +1,18 @@
+import jakarta.servlet.ServletException;
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+
+@WebServlet("/user/delete")
+public class ServletDelete extends HttpServlet {
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        UserDao userDao = new UserDao();
+        String id = request.getParameter("id");
```

Comment:
CLEAN-CODE: lze zjednodusit na jeden radek.

```suggestion
userDao.delete(Integer.parseInt(request.getParameter("id")));
```